libcontainer/cgroups/utils_test.go

```diff
@@ -487,25 +485,29 @@ func TestGetHugePageSizeImpl(t *testing.T) {
 		{ // invalid prefix (silently skipped)
 			input: []string{"whatever-1024kB"},
 		},
-		{ // invalid suffix (skipped with a warning)
-			input:  []string{"hugepages-1024gB"},
-			isWarn: true,
+		{ // invalid suffix
+			input: []string{"hugepages-1024gB"},
+			isErr: true,
+		},
+		{ // no suffix
+			input: []string{"hugepages-1024"},
+			isErr: true,
 		},
-		{ // no suffix (skipped with a warning)
-			input:  []string{"hugepages-1024"},
-			isWarn: true,
+		{ // mixed valid and invalid entries
+			input:  []string{"hugepages-4194304kB", "hugepages-2048kB", "hugepages-akB", "hugepages-64kB"},
+			output: []string{"4GB", "2MB", "64KB"},
+			isErr:  true,
+		},
+		{ // mixed valid and invalid entries
```

Comment:
No need to change, but could include the comment as a `doc: "mixed valid and invalid entries",`, and then use subtests;

```
for _, c := range testCases {
    c :=c
    t.Run(c.doc, func(t *testing.T) {
...
```

That way the test output is documenting what test fails (easier to find back which test is failing), (obvious disclaimer that I like subtests for that reason 😅)

Reply:
Makes sense, added (as a separate commit for clarity and so you don't have to re-review the whole set).